src/redux/services/fileEntry.ts

```diff
@@ -107,7 +108,8 @@ export function readFiles(
       if (fileIsExcluded(appConfig, fileEntry)) {
         fileEntry.isExcluded = true;
       } else if (getFileStats(filePath)?.isDirectory()) {
-        if (depth === appConfig.folderReadsMaxDepth) {
+        const folderReadsMaxDepth = appConfig.projectConfig?.folderReadsMaxDepth || appConfig.folderReadsMaxDepth;
```

Comment:
shouldn't the `projectConfig` already have a default value for this? (and the other properties...)

Reply:
I will explain in a comment at bottom